raiden_contracts/tests/fixtures/secret_registry.py

```diff
@@ -1,16 +1,6 @@
 import pytest
-from raiden_contracts.utils.config import C_SECRET_REGISTRY
 
 
 @pytest.fixture()
-def get_secret_registry(chain, create_contract):
-    def get(arguments, transaction=None):
-        SecretRegistry = chain.provider.get_contract_factory(C_SECRET_REGISTRY)
-        contract = create_contract(SecretRegistry, arguments, transaction)
-        return contract
-    return get
-
-
-@pytest.fixture()
-def secret_registry(get_secret_registry):
-    return get_secret_registry([])
+def secret_registry(secret_registry_contract):
+    return secret_registry_contract
```

Comment:
Where is `secret_registry_contract` defined?

Reply:
it's part of fixtures defined in `raiden-libs` and used for testing in PFS & MS.

https://github.com/raiden-network/raiden-libs/blob/master/raiden_libs/test/fixtures/contracts.py#L93

(well it'd probably make more sense to remove this redefinition and use original fixture instead)

Reply:
Can also be done in another PR. I don't have anything against renaming it to `secret_registry_contract` though.